assets/base.css

```diff
@@ -1550,8 +1550,9 @@ input[type='checkbox'] {
 
 /* component-quantity */
 .quantity {
-  border: 0.1rem solid rgba(var(--color-foreground), 0.55);
+  border: 0.1rem solid rgba(var(--color-base-text), 0.08);
```

Comment:
Related to @Guillaumegranger1 answer here https://github.com/Shopify/dawn/pull/393#issuecomment-903829793 we'll keep this opacity and rely on the visual affordance from the icons which indicate “this is a thing” because they retain the 3:1 ratio. We’ll need to run it by Fable for extra feedback moving forward to iterate if required.